hack/log/log-dump.sh

```diff
@@ -127,8 +127,52 @@ dump_workload_cluster_logs() {
     wait
 }
 
+dump_workload_cluster_logs_windows() {
+    echo "Deploying log-dump-daemonset-windows"
+    "${KUBECTL}" apply -f "${REPO_ROOT}/hack/log/log-dump-daemonset-windows.yaml"
+    echo "Waiting for log-dump-daemonset-windows"
+    "${KUBECTL}" wait pod -l app=log-dump-node-windows --for=condition=Ready --timeout=5m
+
+    IFS=" " read -ra log_dump_pods <<< "$(kubectl get pod -l app=log-dump-node-windows -ojsonpath='{.items[*].metadata.name}')"
+
+    for log_dump_pod in "${log_dump_pods[@]}"; do
+        local node_name
+        node_name="$(get_node_name "${log_dump_pod}")"
+        echo "Getting logs for node ${node_name}"
+
+        local log_dump_dir="${ARTIFACTS}/workload-cluster/${node_name}"
+        mkdir -p "${log_dump_dir}"
+
+        # make a new folder to copy logs to since files cannot be read to directly
+        "${KUBECTL}" exec "${log_dump_pod}" -- cmd.exe /c mkdir l 
```

Comment:
any reason this is just `l`?  can we call it `log` to make it clearer, in particular in the line below (https://github.com/kubernetes-sigs/cluster-api-provider-azure/pull/2043/files#diff-c9e8c42c5a410aeedc20f4f5705d48066d58a8a47b0090509e32d8b10475f952R155) it is hard to read with the default font

Reply:
Yea, no problem.

Reply:
Renamed.